ext/datasource/etcdv3/etcdconfig.go

```diff
@@ -6,10 +6,10 @@ import (
 )
 
 const(
-	EndPoints = "endpoints"
-	User = "user"
-	PassWord = "password"
-	AuthEnable = "enable"
+	EndPoints = "csp.sentinel.etcd.endpoint"
+	User = "csp.sentinel.etcd.user"
+	PassWord = "csp.sentinel.etcd.password"
+	AuthEnable = "csp.sentinel.etcd.auth.enable"
```

Comment:
> Maybe not a good idea to reuse core/config.
> For example, etcd supports SSL, but the current data source configuration does not.

Do you have any suggestion about config?
I think reuse core/config might be okay, but we should to support SSL.

Reply:
Different data sources have different config structs, so just let it should be.
I think it may be better to Pass etcdve.Config to NewEtcdv3DataSource.

Reply:
BTW, I think core should note have too much to extension.

Reply:
Passing etcdv3.Config into NewEtcdv3DataSource is a good idea.
But I think we also should reuse the core/config to store the config parameters, and then use these config parameters to structure the etcdv3.Config.

Reply:
Keep the existing code and add parameters. After deciding how to pass parameters, refact code.

Reply:
remove the configuration file. Now user should provide the etcdv3.Config